srv/salt/ceph/ganesha/restart/controlled/default.sls

```diff
@@ -1,20 +1,17 @@
-{% if salt['cephprocesses.need_restart'](role='ganesha') == True %}
-
 restart nfs-ganesha:
-  module.run:
-    - name: service.restart
-    - m_name: nfs-ganesha
+  deepsea.state_apply_if:
```

Comment:
What's the reasoning behind changing the current theme?

Reply:
when running the stage using deepsea-cli the the restart check made in jinja runs before the code that actually sets the grain value, which causes deepsea-cli to think that that state will not be executed.

This changes moves the check to the "runtime" phase instread of the "parsing" phase.

Reply:
this makes sense, as discussed. We should however be consistent across our codebase and adapt all restart/* to the new implementation.

Reply:
Yes, I agree